<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPRConfigWidget</class>
 <widget class="QWidget" name="MPRConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>123</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>123</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MPR</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_cuttingPlanes">
     <property name="text">
      <string>Cutting Planes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_xPlane">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_xPlane">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox_yPlane">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_yPlane">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox_zPlane">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_zPlane">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
